flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskManagerTest.java

```diff
@@ -486,7 +489,8 @@ protected void run() {
 		public void onReceive(Object message) throws Exception {
 			if(message instanceof RegistrationMessages.RegisterTaskManager){
 				final InstanceID iid = new InstanceID();
-				getSender().tell(new RegistrationMessages.AcknowledgeRegistration(iid, -1),
+				getSender().tell(new RegistrationMessages.AcknowledgeRegistration(iid, -1,
+								Option.<ActorRef>apply(null)),
```

Comment:
That was the only way I found to create a ```None``` with type parameter ```ActorRef``` in Java. ```None``` itself in Java has no type parameter.